docs-source/content/userguide/managing-domains/model-in-image/runtime-updates.md

```diff
@@ -226,3 +232,277 @@ For example, assuming you've installed WDT in `/u01/wdt/weblogic-deploy` and ass
+
+ - Modifying contents of the secrets that are referenced by WDT model(s) and/or modifying the contents of `domain.spec.configuration.model.configMap`.  You will update them in the same manner as for offline updates.
+ - Setting `domain.spec.configuration.model.onlineUpdate.enabled` to `true`
+ - Setting `domain.spec.configuration.model.onlineUpdate.onNonDynmaicChanges` to the desired behavior for non-dynamic changes
+ - Changing `domain.spec.configuration.introspectVersion`.
+
+For any non-dynamic WebLogic mbean attribute added or changed, it requires a restart of the domain.  The Operator's default behavior is to commit all changes and let the user manually restart the domain when ready.  You can control
+this behavior by setting the attribute `domain.spec.configuration.model.onlineUpdate.onNonDynamicChanges` to one of these values: 
+    
+ | OnNonDynamicChanges Value | Behavior | 
+    |---------------------|-------------|
+    | CommitUpdateOnly | Default or if not set.  All changes are committed, but if there are non-dynamic mbean changes. The domain needs to restart manually. |
+    | CommitUpdateAndRoll |  All changes are committed, but if there are non-dynamic mbean changes, the domain will rolling restart automatically; if not, no restart is necessary |
+    | CancelUpdate | If there are non-dynamic mbean changes, all changes are canceled before they are committed. The domain will continue to run, but changes to the configmap and resources in the domain resource YAML should be reverted manually, otherwise in the next introspection will still use the same content in the changed configmap |
+
+Sample domain resource YAML:
+
+```
+apiVersion: "weblogic.oracle/v8"
+kind: Domain
+metadata:
+  name: sample-domain1
+  namespace: sample-domain1-ns
+  labels:
+    weblogic.resourceVersion: domain-v2
+    weblogic.domainUID: sample-domain1
+spec:
+  ...
+  introspectVersion: 5
+  configuration:
+    ...
+    model:
+    ...
+      onlineUpdate:
+        # enable the update to use dynamic update method which does not require restart for dynamic attributes
+        enabled: true
+        # Control the restart behavior of the domain when non-dynamic changes are involved in the onlineupdate
+        # 
+        # CommitUpdateOnly (Default):  All changes are committed. If the changes involve non-dynamic mbean attributes 
+        #                              User has to restart the domain manually, use `kubectl describe domain <domain uid> to view the condition afater the update.
+        # CommitUpdateAndRoll:   All changes are committed. If the changes involve non-dynamic mbean attribues, the Operator will restart the domain automatically.
+        # CancelChanges:  If the changes involve non-dynamic mbean attributes, no changes will be applied, the introspector job will not retry.  User should revert the changes to the config map or resources manually.
```

Comment:
Is this CancelUpdate?

Reply:
Please hold off on reviewing runtime-updates.md - we're working on a rewrite.

Reply:
Yes, it should be CancelUpdate

Reply:
Cancel has been removed. Closing comment as no longer applicable.